server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -53,13 +58,34 @@ sed -Ei \
     ${PB_DEPLOY_FILES}/pbench-server.cfg
 
 # Set up the /srv/pbench file tree
+podman unshare rm -rf ${SRV_PBENCH}/*
```

Comment:
Do we really want to be blowing away `/srv/pbench/archive` every time we deploy a new server container? (I suspect the answer is ... "no".)

Reply:
This is an interesting question.
- For the Staging server, this code is never run.
- For the CI, I think we want to start with a known state which probably is an empty file tree (at least, that's the most convenient).
- For the developer running a canned server for debugging, perhaps we would like the state to persist between runs...I'm not sure.

So, the answers are:
- "doesn't matter";
- "yes"; and,
- "I don't know".

Which, at least at the moment, doesn't sound like "no", to me.  😁

Reply:
Ah. Fair enough... indeed, the staging server was on my mind and that's out of scope. For the CI, resetting is good. Right.

Reply:
😌